bokeh/models/tiles.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import absolute_import
 from ..plot_object import PlotObject
-
-from ..properties import (Any, Dict, Float, String, Int)
+from ..properties import (Any, Dict, Float, String, Int, Bool)
 
 class TileSource(PlotObject):
```

Comment:
> This branch has a critical tile renderer fix (...)
>>@brendancol, as you used the word fix, I would like to see how I can evaluate that this PR is actually a fix? I presume running examples is sufficient. What should I observe? Did you consider adding a test/benchmark for this?

This fixes the scenario where other bokeh elements call `request_render()`.  This was triggering duplicate requests for tiles pending load. In some cases, tile would be requests 4-5 times.  Simple fix was a `loaded` flag on tiles. 

The specific use case I ran into was with the `map sandwhich` style:
&uarr;`tiled label layer`
&uarr;`dynamic data layer` (e.g. datashader)
&uarr;`tiled base layer`

It was here that demonstrates duplicate tile requests between layers. 
Actually sounds like a decent bokeh example. Show I go ahead and add it?


 ( base layer  data layer /